.github/ISSUE_TEMPLATE/bug-report---.md

```diff
@@ -0,0 +1,25 @@
+---
+name: "Bug report \U0001F41B"
+about: Found an error or some unexpected behavior? Create a report of a bug
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+Browser & OS (if known): <Browser>:<OS>
```

Comment:
Maybe we can link to something like https://www.whatismybrowser.com/ to help users share exact versions